po/es.po

```diff
@@ -947,12 +940,12 @@ msgstr "Artículo guardado en %s"
 #: src/itemlistformaction.cpp:1338 src/itemviewformaction.cpp:487
 #, c-format
 msgid "Error: couldn't save article to %s"
-msgstr "Error: no se pudo guardar el artículo como %s."
+msgstr "Error: no se pudo guardar el artículo como %s"
 
 #: src/itemlistformaction.cpp:1423
 #, c-format
 msgid "Search Result - '%s'"
-msgstr "Resultado de la búsqueda - «%s»"
+msgstr "Resultado de la búsqueda - '%s'"
```

Comment:
Sorry for wading into your territory, but this looks inconsistent: you're using angle quotes for the search string on line 687, but here you use single quotes.

Please ignore this comment if there's an actual rationale behind this.